src/Uno.UWP/WinRTFeatureConfiguration.MessageDialog.cs

```diff
@@ -11,6 +11,18 @@ public static class MessageDialog
 		/// Note the native dialogs may not support all the features and they are also not
 		/// supported on Skia targets.
 		/// </summary>
-		public static bool UseNativeDialog { get; set; } = false;
+		public static bool UseNativeDialog { get; set; } =
+			#if __ANDROID__ || __IOS__ || __MACOS__
```

Comment:
```suggestion
			#if __ANDROID__ || __IOS__ || __MACOS__ || __CATALYST__
```

Reply:
`__CATALYST__` needs to be specified? Because then I would expect in many places where we have `__IOS__` or `__MACOS__` only that could cause issue 🤔

Reply:
Actually, you're right. `__IOS__` is defined also when `__CATALYST__` is defined.